src/board/devices/output/leds.ts

```diff
@@ -1,13 +1,32 @@
 import { Device } from 'board/devices/device'
-import { Word } from 'types/binary'
+import { Byte, Word } from 'types/binary'
 
 export class LEDs extends Device {
-  public startAddress = Word.fromUnsignedInteger(0x08000000)
-  public endAddress = Word.fromUnsignedInteger(0x081fffff)
+  public startAddress = Word.fromUnsignedInteger(0x60000100)
+  public endAddress = Word.fromUnsignedInteger(0x60000103)
   public isReadOnly = false
   public isVolatile = false
```

Comment:
Should probably be `true`. I think LED's lose their state when the board is reset.